<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMultiStateGenelCunstomDlg</class>
 <widget class="QDialog" name="QMultiStateGenelCunstomDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>251</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>S#</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>值</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>370</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancle">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_up">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>往上移</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_down">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>往下移</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sData">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>设定值：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
